Packages/com.unity.inputsystem/CHANGELOG.md

```diff
@@ -46,6 +46,7 @@ however, it has to be formatted properly to pass verification tests.
 - `PlayerInput` no longer logs an error message when it is set to `Invoke UnityEvents` and can't find  an action in the given `.inputactions` asset ([case 1259577](https://issuetracker.unity3d.com/issues/an-error-is-thrown-when-deleting-an-input-action-and-entering-play-mode)).
 - Fixed `HoldInteraction` getting stuck when hold and release happens in same event ([case 1346786](https://issuetracker.unity3d.com/issues/input-system-the-canceled-event-is-not-fired-when-clicking-a-button-for-a-precise-amount-of-time)).
 - Fixed adding an action in the `.inputactions` editor automatically duplicating interactions and processors from the first action in the map.
+- Fixed incorrect indexing in `InputActionsSetupExtensions.ChangeBinding`. Contribution by [Fredrik Ludvigsen](https://github.com/steinbitglis) in [#1348](https://github.com/Unity-Technologies/InputSystem/pull/1352).
```

Comment:
`incorrect indexing` is rather vague. Suggest something like
```
* Fixed `InputActionSetupExtensions.ChangeBinding` modifying binding from a different action than specified. ...
```

Reply:
This have now been updated similar to suggestion.